src/main.rs

```diff
@@ -83,6 +83,7 @@ arg_enum!{
     #[allow(non_camel_case_types)]
     pub enum OutputFormat {
         flamegraph,
+        flamegraph_stack_lines,
```

Comment:
Struggled with naming here because the original project was kind of fuzzy about what it called this state. Maybe flamegraph_raw or raw_flamegraph is better?

Reply:
I think a name with "raw" makes sense. flamegraph_folded also seems reasonable.

Reply:
Hey, I was wanting to work on adding this to rbspy and found out that this PR exists. Thanks :) 
I believe that a more suitable naming for this would be `collapsed` or `raw-collapsed`